en/docs/Learn/DesignAPI/CreateAPI/expose-a-soap-service-as-a-rest-api.md

```diff
@@ -1,63 +1,67 @@
+    
+     The created API appears in the Publisher as follows.
 
-4.  The created API appears in the publisher as follows.
-    ![](../../../assets/img/Learn/created-soap-api.jpg)
+     ![]({{base_path}}/assets/img/Learn/created-soap-api.jpg)
+     
+4. Click **API Definiton** to view the API definiton of the created schema.
 
-5.  API definiton of the Created schema has been displayed at **API Definiton** tab.
-    ![](../../../assets/img/Learn/api-definition-of-soap-api-created-by-passthrough-mode.jpg)
+    ![]({{base_path}}/assets/img/Learn/api-definition-of-soap-api-created-by-passthrough-mode.jpg)
   
-    <html><div class="admonition note"><p class="admonition-title">Note</p>
+     <html><div class="admonition note"><p class="admonition-title">Note</p>
      <p>
-            If you wish to add scopes to the resources that were created, navigate to  ***Resources***  and expand the resources. Thereafter, creating new scopes and specify them under operation scope. If you specify a scope, you need to use the same scope when generating access tokens for the subscribed application to invoke the API. For more information on working with the scopes, see
-    [OAuthscopes](../../../../Learn/APISecurity/OAuth2/OAuth2Scopes/fine-grained-access-control-with-oauth-scopes/)
-            </p>
-        </div></html>   
+     If you wish to add scopes to the resources that were created, click  **Resources**. Thereafter, create new scopes and specify them under operation scope. If you specify a scope, you need to use the same scope when generating access tokens for the subscribed application to invoke the API. For more information on working with the scopes, see [OAuthscopes]({{base_path}}/Learn/APISecurity/OAuth2/OAuth2Scopes/fine-grained-access-control-with-oauth-scopes/).
+     </p>
+     </div></html>   
+
+     ![]({{base_path}}/assets/img/Learn/add-scope-for-passthrough-soap-api.jpg)
 
-    ![](../../../assets/img/Learn/add-scope-for-passthrough-soap-api.jpg)
      <html><div class="admonition note">
      <p class="admonition-title">Note</p>
-     <p> Note that when creating this API, the default option of **Rate limiting level** , was selected to **API Level**. For more information on setting advanced throttling policies,
-     see [Enforce Throttling and Resource Access Policies](../../../../Learn/RateLimiting/setting-throttling-limits/).</p>
+     <p> Note that when creating this API, **API Level** was selected as the default option for the **Rate limiting level**. For more information on setting advanced throttling policies,
+     see [Enforce Throttling and Resource Access Policies]({{base_path}}/Learn/RateLimiting/setting-throttling-limits/).</p>
      </div>
      </html>
 
-6. Now, the SOAP service as a RESTful API created and configured successfully. Refer [Publish API](../../../../Learn/DesignAPI/PublishAPI/publish-an-api/) to know details of publishing API.
+Now, the SOAP service are created and configured successfully as a RESTful API. 
```

Comment:
SOAP service are created and configured  should be corrected as **SOAP service is created and configured**